Localization/Translations/Keybindings.lua

```diff
@@ -0,0 +1,21 @@
+---@type l10n
+local l10n = QuestieLoader:ImportModule("l10n")
+
+local keybindingsLocales = {
+    ["Toggle Questie Journey"] = {
```

Comment:
Re-asking if we can use the existing "Toggle My Journey" translation?

https://github.com/Questie/Questie/blob/b27363f8aac0f9b247766f92d02b915402d006e2/Localization/Translations/MinimapIcon/MinimapIcon.lua#L29